.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+    "presets": [
+        "@babel/preset-env"
```

Comment:
You can specify current node as a target to not transpile things that are supported by v8.